easybuild/easyconfigs/a/ABINIT/ABINIT-7.6.2-foss-2015a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "ConfigureMake"
+
+name = 'ABINIT'
+version = '7.6.2'
+
+homepage = 'http://www.abinit.org/'
+description = """Abinit is a plane wave pseudopotential code for doing
+ condensed phase electronic structure calculations using DFT."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
```

Comment:
no, the extra spaces at the end of several lines (see also https://github.com/AFCJamie/easybuild-easyconfigs/pull/1/files)

sorry to be so picky, but this really gets in the way of being able to easily review pull requests using `eb --review-pr`